consensus/logical/message_processor_group.go

```diff
@@ -218,6 +222,7 @@ func (p *Processor) OnMessageGroupInit(msg *model.ConsensusGroupRawMessage) {
 	tlog := newHashTraceLog("OMGI", gHash, msg.SI.GetID())
 
 	if msg.SI.DataHash != msg.GenHash() || gh.Hash != gh.GenHash() {
+		// hold it for now
 		panic("msg gis hash diff")
```

Comment:
return directly replace of panic for illegal user input

Reply:
done